.github/pull_request_template.md

```diff
@@ -25,8 +25,7 @@ Applicable spec: <link>
 - [ ] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
 - [ ] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
 - [ ] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
-- [ ] The documentation is generated using `src-docs`
 - [ ] The documentation for charmhub is updated.
 - [ ] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)
-
+- [ ] The changelog `docs/changelog.md` is updated.
```

Comment:
It might be good to convert "`docs/changelog.md`" to "[changelog](../docs/changelog.md)".
This would change it to a link that can be clicked.